metrics/storage/fio_job.sh

```diff
@@ -29,15 +29,16 @@ TEST_ARGS="$@"
 # These values are a basic configuration for a FIO job
 # execution, however they could be overridden in order to
 # run a customized FIO job.
-BLOCK_SIZE="4k"
+BLOCK_SIZE="16k"
```

Comment:
I think us setting sensible (meaningful, useful) defaults in this file is fine, but you also need to change the command line arguments that get passed in during the actual CI testing, which come from:
https://github.com/clearcontainers/tests/blob/master/.ci/run_metrics_ci.sh#L134

Reply:
@grahamwhaley done

Reply:
@grahamwhaley one question why in the run_metrics_ci we have
$ bash storage/fio_job.sh -b 16k ....
when the block size is 4k?

Reply:
iirc, we chose it as a middle ground between 'small transfers' and 'large transfers'.
Ideally we'd do a spectrum of block sizes, but:
- that would multiply the run time by `n`
- we could in theory do a mix of block sizes:
    - but if we got worse on only small or big blocks, that would be harder to tell from a mixed run
    - I have a `blogbench` test in flight, which does a mix of sizes and reads/writes - so that may cover the more 'real world' case

Yes, we could do with more storage tests, but they take a long time and are produce noisy results - it is a WIP

Reply:
@grahamwhaley , I am a bit confused, so before my changes if we ran the run_metrics_ci that runs `$bash storage/fio_job.sh -b 16k ..` what is running is a `block size =16 k` but if you ran the `fio_job.sh` it is running with `block size  4k`. My question is you want to have in the `run_metrics_ci` a different block size than `fio_job.sh`?

Reply:
@grahamwhaley what will confused me is that when you run `fio_job.sh` and `run_metrics_ci` the results file has a title with a `block size =16k`

Reply:
The '4k' in the `fio_job.sh` file is just a default - the expectation is that when it is invoked via a higher level script (like `run_metrics_ci`), then that script will explicitly set the block size and other parameters it wants. `fio_job.sh` should be generic, and then the invocation scripts set it up specifically.
So, we can set sensible defaults in `fio_job.sh`, in case anybody does run it directly or does not set up all the parameters, but what really matters is how we set up `run_metrics_ci.sh` to invoke it under the CI system.

The name - ah, I see the `run_metrics_ci.sh` [sets the name explicitly](https://github.com/clearcontainers/tests/blob/master/.ci/run_metrics_ci.sh#L134) using the `-t` option - that may be where the name confusion is coming from.